code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -282,33 +93,23 @@
 
 	for(var/item in holographic_items)
 		derez(item)
-
 	for(var/obj/effect/decal/cleanable/blood/B in linkedholodeck)
 		qdel(B)
-
 	for(var/mob/living/simple_animal/hostile/carp/holocarp/C in linkedholodeck)
 		qdel(C)
-
-	holographic_items = A.copy_contents_to(linkedholodeck , 1, perfect_copy = FALSE)
+	holographic_items = A.copy_contents_to(linkedholodeck , platingRequired = TRUE, perfect_copy = FALSE)
```

Comment:
```suggestion
	holographic_items = A.copy_contents_to(linkedholodeck, TRUE, FALSE)
```

Reply:
purposefully left in the named parameters

Reply:
But why, there are no additional params you are skipping here, which is the point of named params